File: LB-Assignments/Assignment_9/program3.c
#include<stdio.h>

int CountRange(int iNo)
{
    int iCnt = 0;
    int iRem = 0;

    while(iNo != 0)
    {
        iRem = iNo % 10;

        if(iRem > 3 && iRem < 7)
        {
            iCnt++;
        }

        iNo = iNo/ 10;
    }

    return iCnt;

}
int main()
{
    int iValue = 0;
    int iRet = 0;

    printf("Enter Number : ");
    scanf("%d",&iValue);

    iRet = CountRange(iValue);

    printf("%d",iRet);
    
    return 0;
}

File: LB-Assignments/Assignment_22/program3.java
/*
Input : 5
Output : 1  #  2  #  3  #  4  #  4  #  
*/
import java.util.*;

class Logic
{
    int iNo = 0;

    public Logic(int a)
    {
        this.iNo = a;
    }

    void Pattern()
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            System.out.print(iCnt+"\t#\t");
        }
        System.out.println();
    }
}
class program3
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        int iValue =0;

        System.out.println("Enter number : ");
        iValue = sobj.nextInt();

        Logic lobj = new Logic(iValue);

        lobj.Pattern();

    }
}